gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=0.1.5-SNAPSHOT
+VERSION_NAME=0.2.0-alpha-SNAPSHOT
```

Comment:
We will need to update the snapshot version for _tagging_ and snapshot _release_